### PR TITLE
571 scan annotation generate table

### PR DESCRIPTION
**Pull request by doori**
- added proxy/annotation/generate and unittests
- fixed constants table location to get values from ingestion template.
- added NUM_PARTITION as parameter in the ingestion template, updated schema and unittests.

TODO:
- there may be a schema change based on discussion with scholars.

### CODE REVIEW

**Review by aauker — COMMENTED** [review]
A general remark: I suspect that the ingestion processes for OV, Lung and BC might differ as the way the case-annotation mappings are found seems different in each project.  For now I don't think we should worry too much, so long as we can still ingest the annotations to the same schema and downstream applications are consistent.

File: data_processing/common/constants.py [hunk 1]
```diff
@@ -28,9 +28,10 @@
 # Configurations
 APP_CFG				='APP_CFG'
 DATA_CFG			='DATA_CFG'
+SCHEMA_FILE			='data_ingestion_template_schema.yml'
 
 def TABLE_LOCATION(cfg): 
-    return "/data/{0}/tables/{1}".format(  cfg.get_value(name=DATA_CFG, jsonpath='PROJECT'), TABLE_NAME(cfg))
+    return "{0}/tables/{1}".format(cfg.get_value(name=DATA_CFG, jsonpath='LANDING_PATH'), TABLE_NAME(cfg))
```
[the remark this answers is not in the document]
Comment (doori) [reply data_processing/common/constants.py:34]: for ingestion templates paths, i think we were using absolute paths /gpfs/.... The hardcoded /data doesn't work for unittests.

**Comment by doori** [comment]
> A general remark: I suspect that the ingestion processes for OV, Lung and BC might differ as the way the case-annotation mappings are found seems different in each project. For now I don't think we should worry too much, so long as we can still ingest the annotations to the same schema and downstream applications are consistent.

Thanks Andy for the comments! Yes, the annotations are not organized, and the standardization needs to happen before we can establish a ingestion process. For now, the annotations are organized as: scan_annotations/{accession_number}

**Comment by arfathpasha** [comment]
LGTM!

**Pull request merged**